omero/sysadmins/unix/server-installation.txt

```diff
@@ -136,9 +136,8 @@ If possible, install one of the following packages:
 +-----------+---------------------------+
 
 OMERO works with the OpenJDK JRE provided by most systems, or with
-Oracle Java. Version 6 or later is required. Version 7 or later is
-recommended (version 6 is end of line and no longer has updates or
-security support). **OpenJDK version 8 is not currently supported**.
+Oracle Java. Version 7 or later is
+recommended. **OpenJDK version 8 is not currently supported**.
```

Comment:
Either is fine.  I stated it this way because we don't know exactly which OpenJDK8 versions/platforms are affected, so it was safer to say it was unsupported, i.e. use at own risk.